### DISCUSSION TITLE
Problems with copilot (translation, explanation and autocomplete)

### DISCUSSION THREAD

**Discussion by Josroyal** [community/community#17958] [Copilot Conversations]
Copilot was working pretty well for me but suddenly, these last 2 days copilot has been failing so often. Now the translation tool doesn´t translate the whole code , it just gives you the first line of the explanation and that´s it (as you can see in the first pic). Also, when I want to use the autocomplete function ( ctrl + enter), the recommended code it´s mostly wrong or you need to go line by line so you can have the "complete solution" made by copilot (2nd pic). I´d like to clarify that I´ve reinstalled both of the extensions, vscode itself, uninstalled all of my extensions except for the copilot one and I even tried to use its past versions  but nothing seems to solve the problem. I hope someone may give me a response because I´m not really sure what could be causing these failures.
![image](https://user-images.githubusercontent.com/64459744/172072999-834dbe72-2d64-430e-b9af-0c43f68680b2.png)
![image](https://user-images.githubusercontent.com/64459744/172073036-c880616a-77a5-4d2f-96b9-02805482bf95.png)
* Sorry if my vocabulary or grammar is not correct, english isn´t my first language.

**Comment by eaftan** [comment]
Sorry for this, but this has been fixed now. If you haven't already, please update the Copilot Labs extension and let us know whether the problem is still occurring.
